skin-deep.js

```diff
@@ -37,7 +39,9 @@ function shallowRender(element, context) {
     },
     shallowRenderer,
     /* eslint-disable no-underscore-dangle */
-    shallowRenderer._instance._instance
+    Versions.ReactBefore16
+      ? shallowRenderer._instance._instance
+      : shallowRenderer._instance
```

Comment:
This difference can also be found in enzyme:

https://github.com/airbnb/enzyme/blob/16f15e028dfb2935467c4f263698c5d3e45e21ce/packages/enzyme-adapter-react-15/src/ReactFifteenAdapter.js#L181

vs.

https://github.com/airbnb/enzyme/blob/16f15e028dfb2935467c4f263698c5d3e45e21ce/packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js#L229